src/test/js/account_lockout_reset_password_test.js

```diff
@@ -1,67 +1,71 @@
-var TestData = require('./config/test_data');
+const TestData = require('./config/test_data');
+const randomData = require('./shared/random_data');
 
 Feature('When I am locked out of my account, resetting my password unlocks it');
 
-let adminEmail;
-let randomUserLastName;
 let citizenEmail;
+let userFirstNames = [];
+let roleNames = [];
+let serviceNames = [];
 
-const serviceName = 'TEST_SERVICE_' + Date.now();
-const testMailSuffix = '@mailtest.gov.uk';
-const password = "Passw0rdIDAM"
+const serviceName = randomData.getRandomServiceName();
 
 BeforeSuite(async (I) => {
-    randomUserLastName = await I.generateRandomText();
-    adminEmail = 'admin.' + randomUserLastName + testMailSuffix;
-    citizenEmail = 'citizen.' + randomUserLastName + testMailSuffix;
+    const randomUserFirstName = randomData.getRandomUserName();
+    const adminEmail = 'admin.' + randomData.getRandomEmailAddress();
+    citizenEmail = 'citizen.' + randomData.getRandomEmailAddress();
```

Comment:
minor: in future, avoid concatenating string with `+` because they're harder to read
use string interpolation instead:
`` `citizen.${randomData.getRandomEmailAddress()}` ``

Reply:
Yes, it's like that in every class, will remove in future.